Tests/LegacyUnit/FrontEnd/PublishEventTest.php

```diff
@@ -132,11 +135,14 @@ public function renderForValidPublicationHashUnhidesEventWithPublicationHash()
 
         $this->subject->render();
 
+        $connection = $this->getConnectionForTable('tx_seminars_seminars');
+
         self::assertTrue(
-            $this->testingFramework->existsRecord(
+            $connection->count(
+                '*',
                 'tx_seminars_seminars',
-                'uid = ' . $eventUid . ' AND hidden = 0'
-            )
+                ['uid' => $eventUid,  'hidden' => 0]
```

Comment:
`$connection->count()` does automatically add the constraints `hidden = 0` and `deleted = 0` to the where clause. Should we remove this here? 

In case we don't want that automatically added we have switch to QueryBuilder and remove the constraints manually.

Reply:
Yes, here we can use the automatic ignoring of hidden and deleted records.